web/package.json

```diff
@@ -22,7 +24,17 @@
     "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject"
   },
-  "proxy": "http://localhost",
+  "proxy": {
+    "/api": {
+      "target": "http://localhost:8070",
+      "secure": false,
+      "logLevel": "debug",
+      "pathRewrite": {
+        "^/api": ""
+      },
+      "changeOrigin": true
+    }
+  },
```

Comment:
Id like this to go back to 

"proxy": "http://localhost",

if it still works. I think its better to hit nginx from dev since thats a little closer to whats in beta/prod

Reply:
I was running into problems with the redirects when I changed it back.